### ISSUE TITLE
display two same  chart panel at once

### ISSUE THREAD

**Issue by jrlj** [jasongilman/proto-repl-charts#19]
When I run the demo codes, it always open two same chart panel at once, why?

**Comment by jasongilman** [comment]
I've never seen that before. Can you take a look at the Atom developer console for errors? You may also want to try restarting Atom.

**Comment by maxthoursie** [comment]
I'm also getting this.

# Steps to reproduce

- Run
```sh 
lein new repro
cd repro
atom --clear-window-state .
```
- Add deps to `project.clj`
```
[proto-repl "0.3.1"]
[proto-repl-charts "0.3.2"]
```
- Open `src/repro/core.clj`
- Replace contents with:
```clojure
(ns repro.core
  (:require [proto-repl-charts.table]))

(proto-repl-charts.table/table "table" [[1 2] [3 4]])
```
- Start repl with `ctrl+, l`
- Eval with `ctrl+, f`

This gives me two panes with the table:

![screenshot from 2018-05-11 14-14-36](https://user-images.githubusercontent.com/1250807/39923856-b23c647a-5525-11e8-9928-db3df7a3ac60.png)

The dev console is empty.

# Environment

Atom version 1.26.1 x64 on Linux

Enabled community packages:
- proto-repl 1.4.24
- proto-repl-charts 0.4.1
-

**Comment by maxthoursie** [comment]
Well, it seems to be intermittent. I was able to reproduce several times with the above steps, but now, following the same steps, it works. :(

**Comment by maxthoursie** [comment]
TLDR: **Remove `cider-nrepl` and it works.**

After some hours of debugging, i found that I had the `cider-nrepl` enabled in my `.lein/profiles.clj`. Cider adds an extra message (`changed-namespaces`) in the nrepl-reply. This message comes after the `status: done` message, which the _nrepl-client_ used by _proto-repl_ doesn't expect. 

For every message after `done`, it will send the whole list of messages to the callback again. This can probably be considered a bug in the nrepl-client. The relevant code is here:

https://github.com/rksm/node-nrepl-client/blob/master/src/nrepl-client.js#L126

https://github.com/clojure-emacs/cider-nrepl/blob/master/src/cider/nrepl/middleware/track_state.clj#L199
